CHANGELOG/v1.5.7.md

```diff
@@ -0,0 +1,35 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.24.x -> v1.28.x
+- Workload Cluster: v1.22.x -> v1.28.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.5.6
+## :chart_with_upwards_trend: Overview
+- 6 new commits merged
+- 1 bug fixed 🐛
+
+## :bug: Bug Fixes
+- e2e: Test: retry GetOwnerGraph when having certificate issues (#10218)
+
+## :seedling: Others
+- CABPK: Add pod metadata to capbk manager (#10213)
+- Dependency: Bump go version to 1.21.8 (#10247)
+- Release: Improve release-staging build (#10227)
+- Release: Read in dockerfiles from stdin (#10210)
+
+:book: Additionally, there has been 1 contribution to our documentation and book. (#10234) 
+
+## Dependencies
```

Comment:
Should we drop this or add a "No dependency changes in this release" statement?

Reply:
Added!